pkg/cli/job/suspend.go

```diff
@@ -36,14 +37,14 @@ var suspendJobFlags = &suspendFlags{}
 
 // InitSuspendFlags init suspend related flags.
 func InitSuspendFlags(cmd *cobra.Command) {
-	initFlags(cmd, &suspendJobFlags.commonFlags)
+	util.InitFlags(cmd, &suspendJobFlags.CommonFlags)
 
 	cmd.Flags().StringVarP(&suspendJobFlags.Namespace, "namespace", "n", "default", "the namespace of job")
 	cmd.Flags().StringVarP(&suspendJobFlags.JobName, "name", "N", "", "the name of job")
 }
 
 // SuspendJob suspends the job.
-func SuspendJob() error {
+func SuspendJob(ctx context.Context) error {
```

Comment:
Thanks for your contribution！
 A little doubt why this context is added?

Reply:
`cmd *cobra.Command` 
Provided the context ctx, we should use the one provided by cmd instead of using context.TODO(). We should pass the upstream ctx to the downstream.

```go
rootCmd := cobra.Command{
		Use:   "vjobs",
		Short: "view job information",
		Long:  `view information of a job with specified name or jobs from the same namespace`,
		Run: func(cmd *cobra.Command, args []string) {
			util.CheckError(cmd, vjobs.ViewJob(cmd.Context()))
		},
	}

```